### PR TITLE
fix 3.3 cc file

### PR DESCRIPTION
**Pull request by Bettygogo2021**
Signed-off-by: Bettygogo2021 <91529409+Bettygogo2021@users.noreply.github.com>
Patrick, i've updated 3.3 docs. Pls kindly help me check and merge it. 3ks
/assign @patrickluoyu

### CODE REVIEW

File: content/en/docs/v3.3/installing-on-linux/on-premises/install-kubesphere-on-vmware-vsphere.md [hunk 1]
```diff
@@ -422,77 +422,191 @@ spec:
...
+---
 ---
 apiVersion: installer.kubesphere.io/v1alpha1
 kind: ClusterConfiguration
 metadata:
   name: ks-installer
   namespace: kubesphere-system
   labels:
-    version: v3.3.0
+    version: master
```
Comment (rayzhou2017) [inline content/en/docs/v3.3/installing-on-linux/on-premises/install-kubesphere-on-vmware-vsphere.md:433]: Isn't this doc for v3.3.0?

Comment (Bettygogo2021) [reply content/en/docs/v3.3/installing-on-linux/on-premises/install-kubesphere-on-vmware-vsphere.md]: > Isn't this doc for v3.3.0?
My mistake. I referenced an incorrect link. 3ks for review. I‘ve updated the doc.

**Pull request merged**